Library/Homebrew/os/linux/elf.rb

```diff
@@ -68,6 +68,40 @@ def binary_executable?
     elf_type == :executable
   end
 
+  def rpath
+    return @rpath if defined? @rpath
+
+    @rpath = if HOMEBREW_PATCHELF_RB
+      patcher = patchelf_patcher
+      begin
+        begin
+          patcher.runpath
+        rescue PatchELF::MissingTagError
+          patcher.rpath
```

Comment:
What happens when the ELF file contains neither `RUNPATH` nor `RPATH`?

Reply:
It logs an error message and returns nil

Reply:
That's not the correct behaviour. There should be no error message. Can you please fix this issue?

Reply:
:sweat_smile: